### PR TITLE
connect with API

### PR DESCRIPTION
**Pull request by iampatryk**

### CODE REVIEW

**Comment by iampatryk** [comment]
@blazejprzyluski 
Hi, Could you check my code ?
I created two classes, one of them I used only for string with Key API

File: src/main/java/org/example/CoinGeckoAPIUsage.java [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+import java.io.IOException;
+import java.io.InterruptedIOException;
+import java.net.URI;
+import java.net.http.HttpClient;
+import java.net.http.HttpRequest;
+import java.net.http.HttpResponse;
+
+public class CoinGeckoAPIUsage {
+
+    KeyApi API = new KeyApi();
```
Comment (blazejprzyluski) [inline src/main/java/org/example/CoinGeckoAPIUsage.java:12]: private final

File: src/main/java/org/example/CoinGeckoAPIUsage.java [hunk 2]
```diff
@@ -0,0 +1,31 @@
+package org.example;
+
+import java.io.IOException;
+import java.io.InterruptedIOException;
+import java.net.URI;
+import java.net.http.HttpClient;
+import java.net.http.HttpRequest;
+import java.net.http.HttpResponse;
+
+public class CoinGeckoAPIUsage {
```
Comment (blazejprzyluski) [inline src/main/java/org/example/CoinGeckoAPIUsage.java:10]: Please make the naming more descriptive, as what does this class do

File: src/main/java/org/example/CoinGeckoAPIUsage.java [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+
+    KeyApi API = new KeyApi();
+    private final String addressURLToConnectWithCoinGeckoApi = "https://api.coingecko.com/api/v3/ping?x_cg_demo_api_key=";
+    private final HttpClient client = HttpClient.newHttpClient();
+
+    public HttpResponse<String> APIUsage() {
```
Comment (blazejprzyluski) [inline src/main/java/org/example/CoinGeckoAPIUsage.java:16]: Please make the name more descriptive, what is being retrieved?

File: src/main/java/org/example/CoinGeckoAPIUsage.java (hunk 2 above)
Comment (iampatryk) [reply src/main/java/org/example/CoinGeckoAPIUsage.java:10]: I changed the name to ConnectWithAPICoinGecko, now the name is fine?

File: src/main/java/org/example/CoinGeckoAPIUsage.java (hunk 3 above)
Comment (iampatryk) [reply src/main/java/org/example/CoinGeckoAPIUsage.java:16]: at this point we are only checking the connection. I named it CheckAPIServerStatus

**Review by blazejprzyluski — APPROVED** [review]

**Pull request merged**